pages/manage/productions/index.vue

```diff
@@ -66,6 +80,8 @@
 
         isLoadingEvents: true,
         events: [],
+        toBeDeletedEventId: '',
+        toBeDeletedEventName: '',
```

Comment:
can't we avoid this by passing the id to the modal as discussed? 

delete -> open modal with event information and production passed 
confirm -> trigger action back in index with the eventId passed down

Reply:
yes, I could only pass the id down to the modal and then search for the eventById in the modal itself to retrieve the name. but toBeDeletedEventId still would have to be a data prop in index no?

Is this what you mean, or is it something different?

Reply:
No, I don't think it needs to be, we can communicate through events from the modal. But do correct me if I'm oversimplifying it?